src/js/components/IconSelect.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+    <v-select v-model="mutableValue" :options="options" label="title">
+        <template slot="selected-option" slot-scope="selected">
+            <i class="fa-pull-left" :class="selected.title"></i>
+            {{ selected.title }}
+        </template>
+        <template slot="option" slot-scope="option">
+            <i :class="option.title"></i>
+            {{ option.title }}
+        </template>
+    </v-select>
+</template>
+
+<script>
+import Vue from 'vue';
+import VSelect from 'vue-select';
+import json from './icons.json';
```

Comment:
I suggested to change the json file to be only an array. If you keep that structure, you could import only the `icons` attribute

```suggestion
import { icons } from './icons.json';
```